dev/TabView/TabView.cpp

```diff
@@ -350,14 +350,49 @@ void TabView::OnScrollViewerLoaded(const winrt::IInspectable&, const winrt::Rout
 
         auto increaseButton = SharedHelpers::FindInVisualTreeByName(scrollViewer, L"ScrollIncreaseButton").as<winrt::RepeatButton>();
         m_scrollIncreaseClickRevoker = increaseButton.Click(winrt::auto_revoke, { this, &TabView::OnScrollIncreaseClick });
+
+        scrollViewer.ViewChanged({ this, &TabView::OnScrollViewerViewChanged });
     }
 
     UpdateTabWidths();
 }
 
+void TabView::OnScrollViewerViewChanged(winrt::IInspectable const& sender, winrt::ScrollViewerViewChangedEventArgs const& args)
+{
+    UpdateScrollViewerDecreaseAndIncreaseButtonsViewState();
+}
+
+void TabView::UpdateScrollViewerDecreaseAndIncreaseButtonsViewState()
+{
+    if (auto&& scrollViewer = m_scrollViewer.get())
+    {
+        auto decreaseButton = SharedHelpers::FindInVisualTreeByName(scrollViewer, L"ScrollDecreaseButton").as<winrt::RepeatButton>();
+        auto increaseButton = SharedHelpers::FindInVisualTreeByName(scrollViewer, L"ScrollIncreaseButton").as<winrt::RepeatButton>();
```

Comment:
It seem that those button are always the same and do not get unloaded and loaded, so it might be better to get a reference in OnApplyTemplate instead of searching for those buttons every time we have changed the scrolling position.

Reply:
We can save the buttons as private members. I was wondering why we are doing FindInVisualTree instead of GetTemplateChild, but that might be because the button is in a re-templated scrollviewer inside the control template ?

Reply:
I don't see a problem of getting it in OnApplyTemplate. I will make the changes to see if it works as indented and push out changes if it does.

Reply:
@ranjeshj you are right, it is not possible to get from GetTemplateChild since it is in a re-templated scroll-viewer.

Reply:
performance wise, I don't think this proposal is viable, FindInVisualTreeByName is expensive and we can't call it every time the view changes, I hope we are able to find a way to store a pointer to the buttons in a tracker_ref

Reply:
I will save them after ListView loaded so we only do it once.